packages/components/src/navigation/Stack.test.tsx

```diff
@@ -44,37 +44,36 @@ beforeAll(() => {
 describe('stack push and pop tests', () => {
   let texts: string[];
   let stackItems: React.ReactNode[];
-  let stack: ReactWrapper;
+  let rerenderFxn;
 
-  function mountStack(stackCount: number, itemCount = 5) {
+  function renderStack(stackCount: number, itemCount = 5) {
     texts = makeStackItemTexts(itemCount);
     stackItems = texts.map(makeStackItem);
-    stack = mount(<Stack>{stackItems.slice(0, stackCount)}</Stack>);
-  }
-
-  function updateProps(props: Record<string, unknown>) {
-    stack.setProps(props);
-    stack.update();
+    const { rerender } = render(
+      <Stack>{stackItems.slice(0, stackCount)}</Stack>
+    );
+    rerenderFxn = rerender;
```

Comment:
By renaming as `rerender` above, you can assign and destructure in one statement:
```suggestion
    ({ rerender } = render(<Stack>{stackItems.slice(0, stackCount)}</Stack>));
```